wiki/mkdocs-en.yml

```diff
@@ -56,7 +56,7 @@ extra_css:
   - stylesheets/extra.css
 
 nav:
-  - Home: 
+  - Home: cedille
     - index.md
   - Onboarding:
     - 'Onboarding intro': onboarding/index.md
```

Comment:
This change breaks the website. It can stay the way it is in main 
```yml
  - Home: 
    - index.md
  - Onboarding:
    - 'Onboarding intro': onboarding/index.md
```